packages/cloud/src/libraries/tenants.ts

```diff
@@ -183,8 +222,8 @@ export class TenantsLibrary {
 
     return {
       id: tenantId,
-      name: insertedTenant.name,
-      tag: insertedTenant.tag,
+      name: String(insertedTenant.name),
+      tag: String(insertedTenant.tag),
```

Comment:
Why do name and tag need to cast to string? Aren't they string type already?

Reply:
ah, forgot to remove this, this was for debugging use 😓